### ISSUE TITLE
CPS

### ISSUE THREAD

**Issue by Magneticdud** [Magneticdud/Hitomi-fansub#7]
@MBjdrma your sub is perfect! I'm just fixing a bit of punctuation. In Aegisub there's a column named "CPS". It stands for "characters per second"

For example here:

![immagine](https://user-images.githubusercontent.com/2940598/57618205-11b8cc80-7583-11e9-9a9b-fb1fabcd1dab.png)

The sentence "He says he's going on a business trip this weekend, too." is only shown for 2 seconds, so it is "22 CPS". Aegisub marks this in red because that means it's too fast to read.

I'm going to abbreviate it in 
"He's going on a business trip." - because anyway the context is Keiko just received an email and reads it, so it's more concise.

In this way 17 CPS it's more readable

![immagine](https://user-images.githubusercontent.com/2940598/57618349-6c522880-7583-11e9-9e6d-82f8596a04d3.png)

**Comment by MBjdrma** [comment]
Super! Thanks for feedback. 
OK re CPS, that makes sense -- so, aim for under 20 cps? 15 cps?

**Comment by Magneticdud** [comment]
Yes, better under 15, but of course only if there's an easy way to do that - as you can see in my edits in 82feec8 I didn't change too much (it *looks* like I changed a lot of stuff, while it's just my program removed an empty space at the end of each sentence, so github thinks i changed everything)

**Comment by MBjdrma** [comment]
Also, thanks for patience with my complete ignorance about how to do this! I had reached out to other fansub teams before without response, didn't know how to start.
BTW, how do you know which dramas to sub? Meaning, how do you someone else isn't also subbing the same drama?

**Comment by Magneticdud** [comment]
well in this case... 10 years are passed since broadcast and nobody completed it...

there's a forum, called D-addicts, there are a lot of fansubbers. The fansub threads appear only if you login

**Comment by MBjdrma** [comment]
If not obvious because of the time since broadcast - do people announce on a forum like that "I'm going to sub x y z video"?

**Comment by Magneticdud** [comment]
yes, on d-addicts there's exactly that kind of threads

tomorrow i try to finish the episode 52, then when episode 54 is done, can "announce" that week 9 is ready on d-addicts

**Comment by MBjdrma** [comment]
I use a laptop owned by my employer so I am usually very cautious about things like downloading videos...I watch online instead. Some limitations to what I can watch, but I already watch too much!!
OK - so I'm assigned 54, right? And you'll check the results and remove extra spaces?

**Comment by Magneticdud** [comment]
ok

let me know if you need some extra files

**Comment by Magneticdud** [comment]
my email is tipika@email.it (this post will be deleted soon)

**Comment by MBjdrma** [comment]
thanks! I'm boyd.martha@gmail.com (I'll do the same!)

**Comment by MBjdrma** [comment]
How do I get the existing un-edited subs into Aegisub along with the video?

**Comment by Magneticdud** [comment]
First download it by clicking on RAW

![immagine](https://user-images.githubusercontent.com/2940598/57797138-124f8f80-774a-11e9-8014-a100fdac77d7.png)

It opens in the browser, then you can click SAVE from its menu

![immagine](https://user-images.githubusercontent.com/2940598/57797180-28f5e680-774a-11e9-91fe-f4ab44f938d7.png)

then on aegisub you open the video

![immagine](https://user-images.githubusercontent.com/2940598/57797242-432fc480-774a-11e9-8138-dfc80a79e4b1.png)

then  open the subs you just saved from the browser

![immagine](https://user-images.githubusercontent.com/2940598/57797269-5478d100-774a-11e9-86e5-86b36443cf0b.png)

**Comment by MBjdrma** [comment]
I opened in RAW but they didn't save as a subtitles file...my choices for Save As are "web archive" and "page source" - and then Aegisub didn't recognize that as a Subtitles file. 
I'm in Safari browser, do you think that's why? 
For now, I'm going to finish ep54 the way I did ep53. I'll try again next time!

**Comment by Magneticdud** [comment]
weird... it doesn't even offer me the option as "web archive" or "page source"

![immagine](https://user-images.githubusercontent.com/2940598/57859546-74180400-77f3-11e9-8988-8d3c7df718c4.png)

the address is like this, right?

https://raw.githubusercontent.com/Magneticdud/Hitomi-fansub/master/Hitomi-ep054.srt

what if you right click on RAW, the "save destination as..."?

![immagine](https://user-images.githubusercontent.com/2940598/57859398-213e4c80-77f3-11e9-8e53-47dc070341fc.png)

**Comment by MBjdrma** [comment]
When I switched to Chrome browser, the RAW saved as a subtitles file that I could open in Aegisub. BUT I was too awkward using Aegisub, so it was still faster for me to stream the video online and correct subtitles in Github. Aegisub doesn't seem to have the editing shortcuts I'm used to in Word - I kept re-starting the video by mistake, and the timing apparently restarted each time too, got really messy. I'll play w it some more later. What episode/s do you want me to start working on next?

**Comment by Magneticdud** [comment]
up to you, 55 or 56
